client/src/views/fields/array.js

```diff
@@ -82,9 +82,8 @@ define('views/fields/array', ['views/fields/base', 'lib!Selectize'], function (D
                     translatedOptions: this.translatedOptions
                 }, function (view) {
                     view.render();
-                    this.listenToOnce(view, 'add', function (item) {
+                    view.on('add',function (item) {
```

Comment:
I used the event directly from the view object, so after closing no more listining to the old modal, as we may open the modal several times.